### DISCUSSION TITLE
Restoring NVRAM backups

### DISCUSSION THREAD

**Discussion by jefft4** [zwave-js/zwave-js-ui#3041] [Q&A] [answered]
Hi,

How does one restore a NVRAM backup made by zwave-js-ui?  I can't see any option for it in the UI and when I tried to restore one of the .bin files to an identical z-stick using the Aeotec backup/restore tool, the result was no nodes in the mesh, just the stick.

Thanks!

**Comment by robertsLando** [comment, marked as answer]
https://user-images.githubusercontent.com/11502495/231072540-75bc4fde-36ca-4d0c-b182-7d45c52dfa5d.mp4

**Reply by jefft4** [reply]
Thanks!

**Reply by robertsLando** [reply]
u welcome :)
